automation_tools/__init__.py

```diff
@@ -1931,10 +1931,10 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     version = distro_info()[1]
     details = os.environ.get('CAP_SUB_DETAILS')
     if details is not None:
-        cv_name, env, ak_name = [item.strip() for item in details.split(',')]
+        cv_name, envr, ak_name = [item.strip() for item in details.split(',')]
```

Comment:
why not `env_name` to consist with 2 others? or it's used too?
and btw, in `envr`, what does `r` stand for?